.github/workflows/build-docker.yml

```diff
@@ -79,15 +79,19 @@ jobs:
         run: |
           docker run -v "$(pwd):/freeorion" -v "${{ runner.temp }}/ccache:/ccache_dir" -e CCACHE_DIR='/ccache_dir' -w /freeorion/build ${{ steps.prep.outputs.tagged_image }} /freeorion/build/godot --gdnative-generate-json-api /freeorion/build/godot-api.json
       - name: Generate godot API file
-        if: ${{ matrix.image == 'manjaro' }}
+        if: false
```

Comment:
This will be very confusing when you will see this in the code.  Could you add a comment here. 

You could also keep the image check for documentation purposes

```yaml
${{ matrix.image == 'manjaro' }} && false  # Godot3 is not supported
```

Reply:
Do you plan to fix this false in the future? Could we just remove this section?

Reply:
Yes, I hope Manjaro maintainers found out incompatible difference between Godot 4 and Godot 3. I'll add comments.

Reply:
Added comments.